pkg/governance-scripts/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@balancer-labs/v2-governance-scripts",
+  "version": "1.0.0",
+  "description": "Balancer V2 Governance Scripts",
+  "license": "GPL-3.0-only",
+  "homepage": "https://github.com/balancer-labs/balancer-v2-monorepo/tree/master/pkg/governance-scripts#readme",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/balancer-labs/balancer-v2-monorepo.git",
+    "directory": "pkg/governance-scripts"
+  },
+  "bugs": {
+    "url": "https://github.com/balancer-labs/balancer-v2-monorepo/issues"
+  },
+  "files": [
+    "contracts/**/*",
+    "!contracts/test/*"
+  ],
+  "scripts": {
+    "build": "yarn compile",
+    "compile": "hardhat compile && rm -rf artifacts/build-info",
+    "compile:watch": "nodemon --ext sol --exec yarn compile",
+    "lint": "yarn lint:typescript && yarn lint:solidity",
+    "lint:typescript": "eslint . --ext .ts --ignore-path ../../.eslintignore  --max-warnings 0",
+    "lint:solidity": "solhint 'contracts/**/*.sol'",
+    "test": "yarn compile && mocha --extension ts --require hardhat/register --require @balancer-labs/v2-common/setupTests --recursive",
```

Comment:
We should add these tests to the CI file at the top level

Reply:
True

Reply:
I always forget that

Reply:
That's how I remember :grin: we always have these followed up by the 'oops CI was not running' PR